lab_1/main.py

```diff
@@ -25,8 +34,25 @@ def remove_stop_words(tokens: list, stop_words: list) -> list:
     stop_words = ['the', 'is']
     --> ['weather', 'sunny', 'man', 'happy']
     """
-    pass
-
+    #stop_words = ['the', 'is']
+    list = []
+    if ((bool(stop_words) is True) and (bool(tokens) is True)
+            and (type(stop_words) != int) and (type(tokens) != int)
+            and (type(stop_words) != float) and (type(tokens) != float)):
+        for token in tokens:
+            if token not in stop_words:
+                list.append(token)
+        return list
+    elif (bool(stop_words) is False) and (bool(tokens) is False):
```

Comment:
1) these checks should be at the beginning
2) seems like there are too many, can you generalize them?